bin/amber_spec

```diff
@@ -3,8 +3,11 @@
 echo "Running './bin/ameba':"
 ./bin/ameba
 
-echo "\nRunning 'crystal tool --format --check':"
+echo "\nRunning 'crystal tool format --check':"
 crystal tool format --check
 
 echo "\nRunning 'crystal spec':"
 crystal spec
+
```

Comment:
We still need to fix these.  This is throwing an error in selenium driver being used by the template.